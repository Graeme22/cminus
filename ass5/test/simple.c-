// C-F20
main()
{
    x = 111;
    y = 222;
    z = 333;
}
