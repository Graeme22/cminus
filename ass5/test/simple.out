====================================
FILE: simple.c-
ERROR(4): Variable 'x' is not declared.
ERROR(5): Variable 'y' is not declared.
ERROR(6): Variable 'z' is not declared.
Number of warnings: 0
Number of errors: 3
